src/Umbraco.IdentityExtensions/App_Start/UmbracoCustomOwinStartup.cs.pp

```diff
@@ -27,19 +30,24 @@
         {
             //Configure the Identity user manager for use with Umbraco Back office
 
+			var globalSettings = Current.Configs.GetConfig<IGlobalSettings>();
+
             // *** EXPERT: There are several overloads of this method that allow you to specify a custom UserStore or even a custom UserManager!            
             app.ConfigureUserManagerForUmbracoBackOffice(
-                ApplicationContext.Current,
-				//The Umbraco membership provider needs to be specified in order to maintain backwards compatibility with the 
-				// user password formats. The membership provider is not used for authentication, if you require custom logic
-				// to validate the username/password against an external data source you can create create a custom UserManager
-				// and override CheckPasswordAsync
+                Current.Services, 
+                Current.Configs.GetConfig<IContentSection>(),
```

Comment:
In this method I got an error with no overloads taking 4 params, I think it is because from 8.1+ the ConfigureUserManagerForUmbracoBackOffice method requires the UmbracoMapper to be passed in aswell. I added `Current.Mapper` after `Current.Services` and it removed my error 🙂